AVAX-C.md

```diff
@@ -0,0 +1,24 @@
+Below is a list of full node RPC endpoints that must be called to broadcast the transaction for Avalanche C-Chain (AVAX-C).
+
+Construct the API request to the full node:
+
+    AVAX-C Testnet Request URL: https://api.avax-test.network/ext/bc/C/rpc
+
+    AVAX-C Mainnet Request URL: https://api.avax.network/ext/bc/C/rpc
+
+    These are the offical Avalanche full node RPC endpoints: https://docs.avax.network/tooling/rpc-providers
+
+    Method: POST
+
+    Request Body format (find "tx" in your downloaded JSON):
+
+    Make sure to add the prefix 0x to the "tx" from the downloaded JSON if its not part of the tx.
+
+```json
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "method": "eth_sendRawTransaction",
+  "params": ["0x + tx"]
+}
+```
```

Comment:
```suggestion
Broadcast a transaction to the Avalanche C-Chain (AVAX-C) by constructing an API request to a full node RPC endpoint.

```shell
export REQUEST_URL="<REQUEST_URL>"
    # Testnet: https://api.avax-test.network/ext/bc/C/rpc
    # Mainnet: https://api.avax.network/ext/bc/C/rpc
    # Full node RPC endpoints: https://docs.avax.network/tooling/rpc-providers
export ACCESS_TOKEN="<YOUR_ACCESS_TOKEN>"

curl -X POST $REQUEST_URL \
  -H "Content-Type: application/json" \
  -H "Authorization: Bearer $ACCESS_TOKEN" \
  -d '{
    "id": 1,
    "jsonrpc": "2.0",
    "method": "eth_sendRawTransaction",
    "params": ["0x + tx"]
    # find "tx" in your downloaded JSON
    # add the "0x" prefix to the "tx" if its not already present
  }'
```
```